src/controllers/update/confirm.overseas.entity.details.controller.ts

```diff
@@ -45,12 +44,6 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
-    const appData: ApplicationData = getApplicationData(req.session as Session);
-
-    if (!isActiveFeature(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS) && checkEntityReviewRequiresTrusts(appData)) {
```

Comment:
Is it normal that this part of the code has been removed instead of just removing the flag condition?

Reply:
In some cases it would be but the condition is for the use paper page which needs to be removed as part of the work. Also the whole condition is to capture cases where we add trusts but the flag is off, which will no longer occur. Therefore, this condition should be taken as a whole unit to be removed.

Reply:
If UPDATE_TRUSTS_SUBMIT_BY_PAPER_URL is to be removed, I understand now why you removed this block.
Thanks for the clarification.